recipes/swigcsm/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "csmapi" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/USGS-Astrogeology/swigcsm/archive/{{ version }}.tar.gz
+  sha256: 81a6914facb91cf4d5e7156b59eb37d75c598b13d8de467852e43dac69a6bc58
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake >=3.10
+    - swig
+    - python
```

Comment:
`python` and `setuptools` should be on `host`, because `setup.py` is being used

Reply:
Thanks! CI running now. I am still not properly grokking host vs build. Back to the docs for me.

Reply:
@tadeu I have added python back into the build section since the wrapper needs to be able to find Python.h. Should I also keep python in the host section? I have in the current push.

Reply:
Unless `python` is used exclusively for the build, the ideal is to have `python` in the host section, because if something links with an `.so` from Python, it should be from the "host" platform. It's weird that `Python.h` isn't being found, perhaps some configuration is missing?

(Usually something like `-DCMAKE_PREFIX_PATH=${PREFIX} -DCMAKE_INSTALL_PREFIX=${PREFIX}` passed to CMake).

Reply:
:+1: I am going to mess around with this a bit more after I see that my most recent push fixed the test. Building locally with conda no problem. Then I'll go ahead and give the configuration a try and get all these commits squashed down into something clean. Thanks for the help on this!

Reply:
Here is the current cmake command: `cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$PREFIX -DANACONDA_PYTHON_VERBOSE=ON ..`. I am using a find_anaconda_python script inside of CMake to find the anaconda install (or to use the system python if anaconda is not installed). Perhaps this is what is causing the issue? Does the conda-forge team have a particular method they like to see/use for cmake projects that want to find dependencies inside a conda installation?  (This last question is the crux of why our code does not support windows - I haven't extended the aforementioned cmake script.)

Reply:
Hmm, maybe it's that `find_anaconda_python`, and it's getting confused because there are two activated prefixes at build time, the host and the build prefixes. I've taken [a look](https://github.com/conda-forge/staged-recipes/pull/8195/files), it seems to be trying to find the root environment, and the root environment usually shouldn't be touched.

Normally it's not necessary to have a customization like that, the recipe can pass down to CMake the `$PREFIX` variable (when using `conda-build`, or, if outside a recipe, pass `$CONDA_PREFIX` but in an activated environment).